internal/integration/infra_sdk_emitter.go

```diff
@@ -122,11 +120,66 @@ func (e *InfraSdkEmitter) emitSummary(i *integration.Integration, metric Metric,
 		ps.AddQuantile(*q.Quantile, *q.Value)
 	}
 
-	i.HostEntity.AddMetric(ps)
+	return e.addMetricToEntity(i, metric, ps)
+}
+
+func (e *InfraSdkEmitter) addMetricToEntity(i *integration.Integration, metric Metric, m metrics.Metric) error {
```

Comment:
what is the difference between `metric Metric` and `m metrics.Metric`? They could use some more descriptive names

Reply:
I know. there is a *metric* type from POMI and there's a *metric* type from the Infra SDK. I'll add some prefixes to the imports to make it more clear